buildtest/utils/file.py

```diff
@@ -143,3 +143,76 @@ def resolve_path(path):
 
     if os.path.exists(real_path):
         return real_path
+
+
+def read(filepath):
```

Comment:
Oh gosh! So please do _not_ replace the natively defined "read" function with our own version (same for write) - that would be like creating a variable named str or int. Please call this "read_file" and then "write_file"

Reply:
this is fixed now in https://github.com/shahzebsiddiqui/buildtest-framework/commit/1fbe34ca33da9f65698e68e153a7315491a02979